modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/repositories/appcatalog/ApplicationInterfaceRepository.java

```diff
@@ -55,6 +56,9 @@ protected ApplicationInterfaceEntity saveApplicationInterface(
         String applicationInterfaceId = applicationInterfaceDescription.getApplicationInterfaceId();
         Mapper mapper = ObjectMapperSingleton.getInstance();
         ApplicationInterfaceEntity applicationInterfaceEntity = mapper.map(applicationInterfaceDescription, ApplicationInterfaceEntity.class);
+        if (!isApplicationInterfaceExists(applicationInterfaceId))
```

Comment:
It is good to have curly braces {} even though you have one line if statement. That improves the readability of the code